src/align.cpp

```diff
@@ -1638,12 +1636,10 @@ static chunk_t *scan_ib_line(chunk_t *start, bool first_pass)
 
 static void align_log_al(log_sev_t sev, size_t line)
 {
-   size_t idx;
-
    if (log_sev_on(sev))
    {
-      log_fmt(sev, "%s: line %lu, %lu)",
-              __func__, line, cpd.al_cnt);
+      log_fmt(sev, "%s: line %lu, %lu)", __func__, line, cpd.al_cnt);
+      size_t idx;
```

Comment:
Absolutely. I did't do it as I tried to stick to the original code as much as possible. But me too I prefer placing the definition in the for loop. 

I will walk through the code and where it seems appropriate to me, move the loop counter into the loop.